lib/MTT/Test/Run.pm

```diff
@@ -525,6 +525,12 @@ sub _do_run {
     $tmp = $ini->val($section, "treat_timeouts_as_fail");
     $config->{treat_timeouts_as_fail} = $tmp
         if (defined($tmp));
+    $config->{corefile_size_limit} = -1;
+    $tmp = $ini->val($section, "corefile_size_limit");
+    $config->{corefile_size_limit} = $tmp
+        if (defined($tmp));
+
+    MTT::DoCommand::set_corefile_size_limit($config->{corefile_size_limit});
```

Comment:
Actually, I forgot: there's a reason I set it this way.

If you set a max corefile value for one Test Run, but *don't* set it for another, we want to run it again in the section where you *don't* set it so that it gets set back to an infinite core file size.

I didn't actually check `setrlimit` for an error, so it should be ok even if the current environment doesn't support it.